x-pack/plugins/fleet/common/openapi/bundled.yaml

```diff
@@ -1850,8 +1850,12 @@ components:
             - type: number
         is_default:
           type: boolean
+          deprecated: true
         is_default_fleet_server:
           type: boolean
+          deprecated: true
+        has_fleet_server:
+          type: boolean
```

Comment:
cb

Reply:
Whoops, this was a note for me to **c**ome **b**ack to this before I submitted the review. Disregard 🙃